constants.xml

```diff
@@ -29,6 +29,7 @@
         <N_PREFILLED_PER_ACCOUNT>1000000</N_PREFILLED_PER_ACCOUNT>
         <NEW_NODE_POW_DELAY>10</NEW_NODE_POW_DELAY>
         <POST_VIEWCHANGE_BUFFER>10</POST_VIEWCHANGE_BUFFER>
+        <TEST_NET_MODE>false</TEST_NET_MODE>
```

Comment:
I suggest putting this constant value into a different xml node (like `smart_contract`) as it's not numeric value.